### DISCUSSION TITLE
Explanations for configuring the WebGui

### DISCUSSION THREAD

**Discussion by ghost** [jsuto/piler#137] [Q&A] [answered]
Hello Janus,
I'm back with questions about the configuration because I can't find anything about it in your documentation.

How do I create the legal hold now? Do I have to do this for every email address?

Then what is the difference between _"archiving rules"_ and _"retention rules"_ for you? Above all, does this have to be done for every expected email address?

What are the _"domains"_ for? Does that mean that only the registered domains are archived? And those that are not registered are not archived even though the mail server sends them?

Why are the _groups_ needed if all emails go to Piler-SMTP via BCC anyway and there is an auditor? What else should the groups be able to do? Likewise, why are the _users_ needed? I thought they could log in to their emails via IMAP access.

Can you please explain that for all of us here? It is very difficult, even impossible, to guess your thought process on this.

Best regards

Vanessa

**Comment by jsuto** [comment, marked as answer]
Hello Vanessa,

Login as an admin to the gui, and find the legalhold in administration menu. There you may add any emails you want to participate in the legal hold.

"Archiving rules" is a bit misleading, I'll rename it to exclusion rules. By default piler archives everything, but you can add exceptions not to be archived to the 'archiving rules' (soon to be called as 'exclusion rules).

Retention rules let you to override the default retention you specify in piler.conf, based on various settings, eg. from, to, subject, etc. fields.

The domains section allows you to specify you own company domains. As I stated above, piler archives every email, however, with the domains list it can figure out if it's an incoming or outgoing email, and lets you specify the direction of the emails to be searched in the search field.

The groups allows you to grant access to the emails of some users to the group members.

The users are for the builtin admin and auditor, and it's a fallback authentication method. If you have IMAP authentication, then no need to add users to the local user db of piler.
